DSI/globus_gridftp_server_iRODS.cpp

```diff
@@ -1891,171 +2074,24 @@ globus_l_gfs_iRODS_recv(
     globus_gridftp_server_begin_transfer(op, 0, iRODS_handle);
 
     // start number_of_irods_write_threads threads to read from circular buffer and write to iRODS
-    for (int thr_id = 0; thr_id < number_of_irods_write_threads; ++thr_id)
+    for (int thr_id = 1; thr_id < number_of_irods_write_threads; ++thr_id)
```

Comment:
Maybe worth a comment explaining why we start at 1 (because `thr_id` 0 is also a writer, but not part of the `thread_pool`?)

Reply:
done